server/src/pages/shot/controller.js

```diff
@@ -281,4 +283,14 @@ function render() {
   page.render(model);
 }
 
+document.addEventListener("addon-present", (e) => {
+  if (e.detail) {
+    const capabilities = JSON.parse(e.detail);
+    if (capabilities["copy-to-clipboard"]) {
```

Comment:
This prevents the copy button from being displayed to older versions of the addon.